planning/behavior_path_planner/src/scene_module/lane_change/normal.cpp

```diff
@@ -562,6 +561,15 @@ bool NormalLaneChange::getLaneChangePaths(
     *lane_change_parameters_);
 
   candidate_paths->reserve(longitudinal_acc_sampling_values.size() * lateral_acc_sampling_num);
+  const auto is_near_end_of_road = std::invoke([&]() {
+    const auto distance =
+      !is_goal_in_route
+        ? dist_to_end_of_current_lanes
+        : utils::getSignedDistance(getEgoPose(), route_handler.getGoalPose(), original_lanelets);
+
+    return std::abs(distance - lane_change_buffer) < 4.0;
```

Comment:
You can use `min_length_for_turn_signal_activation` in the `lane_change_parameter`. This parameter is actually used to judge if the ego vehicle is close to the end of the lane.

Reply:
Changed in [5f15f6f](https://github.com/autowarefoundation/autoware.universe/pull/4242/commits/5f15f6f4eb871d624c8e3c633dd555ccab080453)